src/Acuminator/Acuminator.Utils/RoslynExtensions/PXFieldAttributes/AttributeInformation.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Acuminator.Analyzers;
+using Microsoft.CodeAnalysis;
+
+namespace Acuminator.Utilities
+{
+	/// <summary>
+	/// Information about the Acumatica field attributes.
+	/// </summary>
+	public class AttributeInformation
```

Comment:
@saloponov we already have helper FieldAttributesRegister. What is different in this helper?
Actually I would expect a different helper which from a given attribute/attribute collection will retrieve an expanded set of attributes including everything declared on aggregator attributes